tests/integrationtest/r/cte.result

```diff
@@ -788,3 +788,11 @@ with cte1 as (select 1), cte2 as (select 2) select * from cte1 union (with cte2
 1
 1
 3
+set tidb_max_chunk_size=32;
```

Comment:
No need to add more test, delete https://github.com/pingcap/tidb/blob/b8515ae354a537ae1cce9ef32c4ca1afe84eea9e/tests/integrationtest/t/executor/cte.test#L1-L3 is enough.